gqcpy/src/ONVBasis/ONVPath_bindings.cpp

```diff
@@ -0,0 +1,106 @@
+// it under the terms of the GNU Lesser General Public License as published by
```

Comment:
Part of the header is missing.


Furthermore, I've done some changes to the C++ code, so the Python bindings still have to updated accordingly, mostly regarding the documentation.